test/integration/statefulset/statefulset_test.go

```diff
@@ -655,42 +656,47 @@ func TestDeletingPodForRollingUpdatePartition(t *testing.T) {
 
 func TestStatefulSetStartOrdinal(t *testing.T) {
 	tests := []struct {
-		ordinals         *appsv1.StatefulSetOrdinals
-		name             string
-		namespace        string
-		replicas         int
-		expectedPodNames []string
+		ordinals            *appsv1.StatefulSetOrdinals
+		name                string
+		namespace           string
+		replicas            int
+		expectedPodOrdinals []string
```

Comment:
```suggestion
		expectedPodIndexes []string
```
nit: to better correlate the naming with the feature name, and the label name